Update installation_guide.rst (#1022)
yeatmanlab/pyAFQ: docs/source/howto/installation_guide.rst

  git clone https://github.com/yeatmanlab/pyAFQ.git

> With both installation methods, you will need to include extensions. Change your working directory into the top-level directory of this repo

Reviewer: ```suggestion
With both installation methods, you can include extensions to the base installation in square brackets. Change your working directory into the top-level directory of this repo
```
